grails-app/migrations/0.9.x/changelog-2024-12-18-1300-create-table-cycle-count-request.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+    <changeSet author="kchelstowski" id="181220241300-0">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="cycle_count_request"/>
+            </not>
+        </preConditions>
+
+        <createTable tableName="cycle_count_request">
+            <column name="id" type="CHAR(38)">
+                <constraints nullable="false" primaryKey="true" />
+            </column>
+            <column name="version" type="BIGINT">
+                <constraints nullable="false" />
+            </column>
+            <column name="facility_id" type="CHAR(38)">
+                <constraints nullable="false" />
+            </column>
+            <column name="product_id" type="CHAR(38)">
+                <constraints nullable="true" />
+            </column>
+            <column name="status" type="CHAR(38)">
```

Comment:
should this be VARCHAR since it's based on an enum?

Reply:
you are right, I've just copied the type from the `id`. Will fix it.